### PR TITLE
fix: Don't crash if contact could not be loaded

### PR DESCRIPTION
**Pull request by ffflorian**
This PR will
* Apply the suggestions from https://github.com/wireapp/node-addressbook/pull/3:
  * Remove unnecessary casting
  * Return empty `Person` if contact could not be loaded

### CODE REVIEW

**Comment by marcoconti83** [comment]
There are a lot of changes in this PR, most of them are formatting, but there are a couple of code lines that changes. It's hard to follow which ones are the relevant ones. In future I would suggest making separate PRs, one for the formatting, one for the fix.

So far I found:
- [here](https://github.com/wireapp/node-addressbook/pull/13/files#diff-c13ca2585e457acbbe617f6ac1ae699aL61) casting
- [here](https://github.com/wireapp/node-addressbook/pull/13/files#diff-c13ca2585e457acbbe617f6ac1ae699aL118) signed to unsigned
- [here](https://github.com/wireapp/node-addressbook/pull/13/files#diff-c13ca2585e457acbbe617f6ac1ae699aR124) and [here](https://github.com/wireapp/node-addressbook/pull/13/files#diff-c13ca2585e457acbbe617f6ac1ae699aL127) making sure that a person exists

Did I miss any other meaningful change?

File: src/cpp/wrapper.cc [hunk 1]
```diff
@@ -14,138 +14,133 @@
...
+#endif
 
-    AddressBook ab;
-    Isolate* isolate = Isolate::GetCurrent();
+  AddressBook ab;
+  Isolate *isolate = Isolate::GetCurrent();
+  Person *person = ab.getContact(index);
+
+  if (person == NULL) {
+    person = new Person();
```
Comment (marcoconti83) [inline src/cpp/wrapper.cc]: Here we are doing an allocation on the heap. When is that memory released?

**Comment by ffflorian** [comment]
I updated the PR to now include only the necessary changes.

**Comment by marcoconti83** [comment]
Thanks for updating!

File: src/cpp/wrapper.cc [hunk 2]
```diff
@@ -110,16 +110,20 @@ NAN_METHOD(GetMe) {
...
 
   AddressBook ab;
   Isolate* isolate = Isolate::GetCurrent();
+  Person* person = ab.getContact(index);
 
   Local<Object> contact = Object::New(isolate);
-  fillPersonObject(isolate, contact, ab.getContact(index));
+
+  if (person != NULL) {
+    fillPersonObject(isolate, contact, person);
```
Comment (c-g-owen) [inline src/cpp/wrapper.cc]: If person is NULL, you will now get a JS object with no members for name etc. Depending on how you want to use it maybe you prefer to make an object with those members set to null/empty string

Comment (ffflorian) [reply src/cpp/wrapper.cc]: I just tried it on Linux with `getContact(0)` and it returns
```js
{ firstName: '', lastName: '', emails: [], numbers: [] }
```

Comment (c-g-owen) [reply src/cpp/wrapper.cc]: That's good then :)
